rust/agama-lib/share/examples/storage-guided.json

```diff
@@ -0,0 +1,63 @@
+{
+  "storage": {
+    "guided": {
+      "target": {
+        "disk": "/dev/vdc"
+      },
+      "boot": {
+        "configure": true,
+        "device": "/dev/vda"
+      },
+      "encryption": {
+        "password": "notsecret",
+        "method": "luks2",
+        "pbkdFunction": "argon2i"
+      },
+      "space": {
+        "policy": "custom",
+        "actions": [
+          { "resize": "/dev/vda" },
+          { "forceDelete": "/dev/vdb1" }
+        ]
+      },
+      "volumes": [
+        {
+          "mount": {
+            "path": "/",
+            "options": ["ro"]
+          },
+          "filesystem": {
+            "btrfs": {
+              "snapshots": true
+            }
+          },
+          "size": [1024, "5 Gib"],
+          "target": "default"
+        },
+        {
+          "mount": {
+            "path": "/home"
+          },
+          "filesystem": "xfs",
+          "size": {
+            "min": "5 GiB",
+            "max": "20 GiB"
+          },
+          "target": {
+            "newVg": "/dev/vda"
+          }
+        },
+        {
+          "mount": {
+            "path": "swap"
+          },
+          "filesystem": "swap",
```

Comment:
is it smart enough that if omited it will use swap fs automatically?

Reply:
The behavior does not depend on the schema. We can decide in the implementation of the proposal what to do:

* Report an error.
* Assume swap fs if no fs is given.
* Assume swap fs if other fs is given.

One thing that could go to the schema is to force `"filesystem": "swap"` if mount is *swap* and vice versa. But, for now, I would not make the schema more complex.